index.js

```diff
@@ -7,13 +7,34 @@ const arrify = require('arrify');
 const globby = require('globby');
 const cpFile = require('cp-file');
 const junk = require('junk');
+const pFilter = require('p-filter');
 const CpyError = require('./cpy-error');
 
 const defaultOptions = {
 	ignoreJunk: true
 };
 
-const preprocessSourcePath = (source, options) => options.cwd ? path.resolve(options.cwd, source) : source;
+class SourceFile {
+	constructor(relativePath, path) {
+		this.path = path;
+		Object.defineProperty(this, 'relativePath', {enumerable: false, value: relativePath});
```

Comment:
`enumerable: false` is the default.

Reply:
Why are you using `defineProperty` and not `this.path`? Why do it at all?

Reply:
To hide if from dumps and such. It's internal field, API users don't need to know about it and they shouldn't use it.

Reply:
Well, nvm. It's public now.